examples/PushRotate/PushRotate.ino

```diff
@@ -0,0 +1,42 @@
+/* PushRotate
+  by K.Abhijeet
+  This example code rotates the servo motor clockwise or anti-clockwise as per button pressed.
+
+  Circuit:-
+    Arduino Uno
+    Micro Servo Motor connected at pin no 9
```

Comment:
```suggestion
    Micro Servo Motor connected at pin 9
```
"no" might be confusing to the user, and the term "pin 9" should be very easy for anyone with even a small amount of experience with Arduino to understand.